database/mssql/scripts/versions/v_41_ddl.sql

```diff
@@ -0,0 +1,250 @@
+   [APP_LAST_UPDATE_USER_DIRECTORY] [nvarchar](30) DEFAULT(user_name()),
+   [CONCURRENCY_CONTROL_NUMBER] [int] NULL,
+   [DB_CREATE_USERID] [varchar](63) NOT NULL,
+   [DB_CREATE_TIMESTAMP] [datetime2](7) NOT NULL,
+   [DB_LAST_UPDATE_USERID] [varchar](63) NOT NULL,
+   [DB_LAST_UPDATE_TIMESTAMP] [datetime2](7) NOT NULL,
+   CONSTRAINT [PK_ORBC_LOA_PERMIT_TYPE_DETAILS] PRIMARY KEY CLUSTERED ([LOA_PERMIT_TYPE_ID] ASC) WITH (
+      PAD_INDEX = OFF,
+      STATISTICS_NORECOMPUTE = OFF,
+      IGNORE_DUP_KEY = OFF,
+      ALLOW_ROW_LOCKS = ON,
+      ALLOW_PAGE_LOCKS = ON
+      ) ON [PRIMARY]
+   ) ON [PRIMARY];
+
+CREATE TABLE [permit].[ORBC_LOA_VEHICLES] (
+   [LOA_VEHICLE_ID] [int] IDENTITY(1,1) NOT NULL,
+   [LOA_ID] [int] NOT NULL,
+   [POWER_UNIT_ID] [bigint],
+   [TRAILER_ID] [bigint],
+   [APP_CREATE_TIMESTAMP] [datetime2](7) DEFAULT(getutcdate()),
+   [APP_CREATE_USERID] [nvarchar](30) DEFAULT(user_name()),
+   [APP_CREATE_USER_GUID] [char](32) NULL,
+   [APP_CREATE_USER_DIRECTORY] [nvarchar](30) DEFAULT(user_name()),
+   [APP_LAST_UPDATE_TIMESTAMP] [datetime2](7) DEFAULT(getutcdate()),
+   [APP_LAST_UPDATE_USERID] [nvarchar](30) DEFAULT(user_name()),
+   [APP_LAST_UPDATE_USER_GUID] [char](32) NULL,
+   [APP_LAST_UPDATE_USER_DIRECTORY] [nvarchar](30) DEFAULT(user_name()),
+   [CONCURRENCY_CONTROL_NUMBER] [int] NULL,
+   [DB_CREATE_USERID] [varchar](63) NOT NULL,
+   [DB_CREATE_TIMESTAMP] [datetime2](7) NOT NULL,
+   [DB_LAST_UPDATE_USERID] [varchar](63) NOT NULL,
+   [DB_LAST_UPDATE_TIMESTAMP] [datetime2](7) NOT NULL,
+   CONSTRAINT [PK_ORBC_LOA_VEHICLES] PRIMARY KEY CLUSTERED ([LOA_VEHICLE_ID] ASC) WITH (
+      PAD_INDEX = OFF,
+      STATISTICS_NORECOMPUTE = OFF,
+      IGNORE_DUP_KEY = OFF,
+      ALLOW_ROW_LOCKS = ON,
+      ALLOW_PAGE_LOCKS = ON
+      ) ON [PRIMARY]
+   ) ON [PRIMARY];
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_DETAILS_DB_CREATE_USERID] DEFAULT(user_name())
+FOR [DB_CREATE_USERID]
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_DETAILS_DB_CREATE_TIMESTAMP] DEFAULT(getutcdate())
+FOR [DB_CREATE_TIMESTAMP]
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_DETAILS_DB_LAST_UPDATE_USERID] DEFAULT(user_name())
+FOR [DB_LAST_UPDATE_USERID]
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD CONSTRAINT [DF_ORBC_LOA_DETAILS_DB_LAST_UPDATE_TIMESTAMP] DEFAULT(getutcdate())
+FOR [DB_LAST_UPDATE_TIMESTAMP]
+
+-- Check Contraints
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] WITH CHECK ADD  CONSTRAINT DK_ORBC_LOA_DETAILS_IS_ACTIVE_VAL CHECK ([IS_ACTIVE] IN ('Y','N'));
+GO
+
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD  CONSTRAINT [DF_ORBC_LOA_DETAILS_REVISION]  DEFAULT ((0)) FOR [REVISION]
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD  CONSTRAINT [DF_ORBC_LOA_DETAILS_PREVIOUS_REVISION]  DEFAULT ((0)) FOR [PREVIOUS_LOA_ID]
+ALTER TABLE [permit].[ORBC_LOA_DETAILS] ADD  CONSTRAINT [DF_ORBC_LOA_DETAILS_ORIGINAL_ID]  DEFAULT ((0)) FOR [ORIGINAL_LOA_ID]
```

Comment:
Revision does not exist anymore. it should not have a default value of 0. should be null if not available.